vpdq/python/tests/test_vpdq_hash.py

```diff
@@ -49,6 +49,21 @@ def test_vpdq_utils():
         assert vpdq.hash_to_hex(feature.hash) == hex_hash
 
 
+def test_error_checking():
+    video_file = Path(f"{video_folder}/{TEST_FILES[0]}.mp4")
+    with pytest.raises(ValueError):
```

Comment:
blocking: This will only catch one exception, and the rest of these will not be executed. Additionally, if you are using messages, use the message regex feature to exactly match the expected exception